src/twisted/protocols/amp.py

```diff
@@ -239,6 +239,7 @@ def trapZero(result):
 
 __all__ = [
     "AMP",
+    "AMPv2",
```

Comment:
This should not be exposed as a separate class, but rather a protocol-version parameter.